cis/validation.py

```diff
@@ -0,0 +1,39 @@
+import json
+import os
+
+from jsonschema import validate as jsonschema_validate
+
+from cis.encryption import decrypt
+
+
+CIS_SCHEMA_JSON = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'schema.json')
+
+
+with open(CIS_SCHEMA_JSON, 'r') as schema_data:
```

Comment:
File always exists. Its in the codebase and its going to be shipped with the package.